supervisor/docker/__init__.py

```diff
@@ -230,3 +230,21 @@ def repair(self) -> None:
             _LOGGER.debug("Networks prune: %s", output)
         except docker.errors.APIError as err:
             _LOGGER.warning("Error for networks prune: %s", err)
+
+    def check_denylist_images(self) -> bool:
+        """Return a boolean if the host has images in the denylist."""
+        denied_images = set()
+        for image in self.images.list():
+            for tag in image.tags:
+                imagename = tag.split(":")[0]
+                if (
+                    imagename in DOCKER_IMAGE_DENYLIST
+                    and imagename not in denied_images
+                ):
+                    denied_images.add(imagename)
+                    _LOGGER.error(
+                        "Found image '%s', this is not supported, remove this from the host!",
+                        imagename,
+                    )
+                    denied_images.add(imagename)
```

Comment:
Words are separated with underscore in variable names.
```suggestion
                image_name = tag.split(":")[0]
                if (
                    image_name in DOCKER_IMAGE_DENYLIST
                    and image_name not in denied_images
                ):
                    denied_images.add(image_name)
                    _LOGGER.error(
                        "Found image '%s', this is not supported, remove this from the host!",
                        image_name,
                    )
                    denied_images.add(image_name)
```